app/lib/search_query_builder.rb

```diff
@@ -191,8 +191,14 @@ def base_reject
   end
 
   def facet_query
-    count_dynamic_facets(facet_params.keys)
-    facet_params.reduce({}) { |query, (k, v)| query.merge("facet_#{k}" => v) }
+    dynamic_facet_params = facets_not_overridden_by_registries(facet_params)
+    count_dynamic_facets(dynamic_facet_params.keys)
```

Comment:
Is it possible that all facet key will always be present? We serialize the search form, but I'm not sure if we strip out any unused params (so if you've not selected organisations you'll still see `&organisations[]=&` in the query params).

Reply:
The facets should be independent of the query string I think?  It should just provide options of additional things to filter by rather than filtering the query itself.

Reply:
Ah yeah you are right, I thought `facet_params` came from the query but it looks like it comes from the content item.